src/server_manager/ui_components/outline-server-view.html

```diff
@@ -464,42 +455,11 @@ <h3>[[accessKeyRows.length]]</h3>
               </span>
               <span class="flex-1 header-row-spacer"></span>
             </div>
-            <!-- admin row -->
-            <div class="access-key-row" id="managerRow">
-              <span class="access-key-container">
-                <img class="manager-access-key-icon access-key-icon" src="images/manager-key-avatar.svg" />
-                <div class="access-key-name">
-                  <div>[[localize('server-my-access-key')]]</div>
```

Comment:
Let's remove the strings that we don't need anymore.

Reply:
👍

Reply:
I think you missed `master_messages.json`.

Reply:
Thanks :)